examples/hotsos-example-mysql.short.summary.yaml

```diff
@@ -0,0 +1,6 @@
+potential-issues:
+  system:
+    SystemWarnings:
+      - Unattended upgrades are enabled which can lead to uncontrolled changes to
```

Comment:
ah right so that's because the system plugin is *always* run so running --mysql is the same as --mysql --system and that scenario message is from the system plugin (i think it actually appears in all the examples as a result).